### PR TITLE
Nonprotobuf

### PR DESCRIPTION
**Pull request by janm399**
This is to compare manual code to decode the message to the protobuf code. It seems to be that there is a lot of messy generated code by protobuf. (It includes both the Java / Scala code as well as the Swift code.)

I'd love to hear your comments.

### CODE REVIEW

File: ios/Lift/LiftServer/SensorDataSourceLocation.swift [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import Foundation
+
+/**
+ * Sensor data location matching the Scala codebase
+ */
+enum SensorDataSourceLocation : UInt8 {
+    case Wrist = 0x01
```
Comment (zapletal-martin) [inline ios/Lift/LiftServer/SensorDataSourceLocation.swift]: This is why I was in favour of shared schema. Now these values are by convention so everyone implementing the encoder/decoder must know what 0x01 means and this code is duplicated in each device's code

File: server/exercise/src/main/scala/com/eigengo/lift/exercise/MultiPacketDecoder.scala [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+ */
+object MultiPacketDecoder {
+  private val header = 0xcab0.toShort
+
+  def decodeShort(b0: Byte, b1: Byte): Int = {
+    ByteVector(b0, b1).toInt(signed = false, ordering = ByteOrdering.BigEndian)
+  }
+
+  def decodeSensorDataSourceLocation(sloc: Byte): String \/ SensorDataSourceLocation = sloc match {
+    case 0x01 ⇒ \/.right(SensorDataSourceLocationWrist)
```
Comment (zapletal-martin) [inline server/exercise/src/main/scala/com/eigengo/lift/exercise/MultiPacketDecoder.scala]: Scalaz offers two more syntexes to do the same (if preferred). 
`\/-(SensorDataSourceLocationWrist)`
and
`SensorDataSourceLocationWrist.right`

File: ios/Lift/LiftServer/SensorDataSourceLocation.swift (hunk 1 above)
Comment (carlpulley) [inline ios/Lift/LiftServer/SensorDataSourceLocation.swift]: @zapletal-martin, I think that's a good point.

File: server/exercise/src/main/scala/com/eigengo/lift/exercise/MultiPacketDecoder.scala (hunk 2 above)
Comment (janm399) [inline server/exercise/src/main/scala/com/eigengo/lift/exercise/MultiPacketDecoder.scala]: I like the \/.right and \/.left... but that's just my preference.

**Pull request merged**